Meadowlark-js/packages/meadowlark-core/src/handler/FrontendFacade.ts

```diff
@@ -48,7 +48,7 @@ function postStack(): MiddlewareStack {
       R.andThen(logRequestBody),
       R.andThen(resourceValidation),
       R.andThen(metaEdModelFinding),
-      R.andThen(documentValidation),
+      R.andThen(documentValidationForInsert),
```

Comment:
You can probably shrink this back down to a single `documentValidation` middleware function.